src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/custom-anomalies.mdx

```diff
@@ -12,90 +12,113 @@ redirects:
 
-Here are some tips for setting anomaly thresholds:
+  To configure the threshold, you'll need to:
 
-* Set the [anomaly direction](#upper-lower) to monitor violations that happen either above or below the anomaly.
-* Set the preview chart to either [2 days or 7 days](#days) of displayed data. This is not applicable for [NRQL alert conditions](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries).
-* Use the slider bar to adjust the  **Critical** threshold sensitivity, represented in the preview chart by the light gray area around the anomaly. The tighter the band around the anomaly, the more sensitive it is and the more violations it will generate.
-* Optional: You can create a [**Warning** threshold](/docs/alerts/new-relic-alerts/defining-conditions/set-thresholds-alert-condition#threshold-levels) (the darker gray area around the anomaly).
-* For NRQL alerts, see the [allowed types of NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#threshold-types).
-* If the alert condition applies to multiple apps, you can select a choice from the dropdown above the chart to use different metrics. (Not applicable for NRQL alert conditions.)
+    * Set the 'threshold direction' to either upper, lower, or both. This means that we'll only create an incident if the signal value (the output of the query) is above the predicted value, below the predicted value, or either, respectively.
 
-### Faceted anomaly conditions [#anomaly-facets]
+    This field answers to how many of the data points during a specified time period must be outside of the threshold. The options are **for at least** and **at least once in**. Selecting **for at least** means that ALL of your signal's data points must be outside of the threshold for the specified time period before an incident is opened. The inverse must be true to close the incident. The **at least once in** option simply means that as soon as any of your signal's data points are outside of the threshold, an incident will open. With this option, the time duration is not relevant for determining when to open an incident. However, it's relevant for closing incidents. All of your signal's data points must be within the threshold for the specified period of time
 
-Once you've run a faceted NRQL query for an anomaly condition, you can scope your results to a single anomaly.
+    * Set the 'threshold duration'. Think of this as how long the signal value must remain outside the threshold before an incident is opened.   Conversely, it's also how long a signal must be within the threshold for an incident to be closed. 
+
+    This field answers to the time period mentioned above. It's how long the signal must be in violation of the threshold being defined. This is the actual threshold duration.
```

Comment:
```suggestion
    This field is defined by the time period mentioned above. It's how long the signal must be in violation of the threshold being defined. This is the actual threshold duration.
```

Reply:
This one I'm not going to add it because it's a passive sentence.